content/tooling/wrangler/configuration/environments.md

```diff
@@ -5,9 +5,9 @@ weight: 3
 
 - [Concepts](#concepts)
 - [Usage](#usage)
-  * [Examples](#examples)
-  * [Custom webpack configurations](#custom-webpack-configurations)
-  * [KV Namespaces with environments](#kv-namespaces-with-environments)
+  - [Examples](#examples)
+  - [Custom webpack configurations](#custom-webpack-configurations)
+  - [Environment Variables](#environment-variables)
```

Comment:
I'd just call this `Environments` to avoid confusion between this and text vars.